src/wizards/tokenSwapDealWizard/stages/primaryDaoStage/primaryDaoStage.ts

```diff
@@ -22,7 +22,9 @@ export class PrimaryDaoStage implements IBaseWizardStage {
     this.disabled = stageMeta.wizardType === WizardType.makeAnOffer;
     this.isPartneredDeal = this.getIsPartneredDeal(stageMeta.wizardType);
 
-    const validationRules = daoStageValidationRules("Primary DAO", this.wizardState.registrationData.partnerDAO);
+    const partnerDao = this.isPartneredDeal ? this.wizardState.registrationData.partnerDAO : null;
```

Comment:
```suggestion
    const partnerDao = this.isPartneredDeal ? this.wizardState.registrationData.partnerDAO : undefined;
```

Because the type `IDealRegistrationTokenSwap` does not specify `null`
`partnerDAO?: IDAO; // This doesn't exist for an Open Proposal`